pkg/orchestrator/util.go

```diff
@@ -33,19 +33,22 @@ const (
 	K8sCronJob
 	// K8sDaemonSet represents a Kubernetes DaemonSet
 	K8sDaemonSet
+	// K8sStatefulSet represents a Kubernetes StatefulSet
+	K8sStatefulSet
 )
 
 var (
 	telemetryTags = map[NodeType][]string{
-		K8sCluster:    getTelemetryTags(K8sCluster),
-		K8sCronJob:    getTelemetryTags(K8sCronJob),
-		K8sDeployment: getTelemetryTags(K8sDeployment),
-		K8sJob:        getTelemetryTags(K8sJob),
-		K8sNode:       getTelemetryTags(K8sNode),
-		K8sPod:        getTelemetryTags(K8sPod),
-		K8sReplicaSet: getTelemetryTags(K8sReplicaSet),
-		K8sService:    getTelemetryTags(K8sService),
-		K8sDaemonSet:  getTelemetryTags(K8sDaemonSet),
+		K8sCluster:     getTelemetryTags(K8sCluster),
```

Comment:
WDYT about refactoring those to build the map dynamically by iterating through the iota (if needed we might need to refactor/add a function to be able to do this)?